src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -1202,6 +1209,18 @@ class DocBaseViewer extends BaseViewer {
                 this.resize();
             }
         }
+
+        // Fire rendered metric to indicate that the specific page of content the user requested has been shown
+        if (!this.startPageRendered && (this.startPageNum === pageNumber || this.getCachedPage() === pageNumber)) {
+            const pageRenderTag = Timer.createTag(this.options.file.id, RENDER_METRIC);
+            const pageRenderTimer = Timer.stop(pageRenderTag) || {};
```

Comment:
Should we still emit the event even if `pageRenderTimer` is an empty object?

Reply:
Fair enough, I can add a guard clause against it.